addons/hr_timesheet/static/tests/timesheet_uom_tests.js

```diff
@@ -0,0 +1,94 @@
+/** @odoo-module */
+
+import { session } from "@web/session";
+
+import { makeView } from "@web/../tests/views/helpers";
+import { getFixture, patchWithCleanup } from "@web/../tests/helpers/utils";
+
+import { getServerData, updateArch, setupTestEnv } from "./hr_timesheet_common_tests";
+
+
+QUnit.module("hr_timesheet", (hooks) => {
+    let target;
+    let serverData;
```

Comment:
I didn't know we could do this. I thought we have to create global variable for the file. 😅 
It is good to know for the next time. 🙂